environments/backup-production/app-processes.yml

```diff
@@ -20,6 +20,7 @@ celery_processes:
     celery,export_download_queue,reminder_rule_queue,case_import_queue,case_rule_queue:
       concurrency: 1
     flower: {}
+    beat: {}
```

Comment:
We actually specifically do _not_ want this to ever be running on the backup. Probably worth leaving a comment about that. That is because celery beat triggers scheduled events like sending SMS's and such, and we don't want those to get kicked off on the backup environment when we restore the data.

Reply:
Same reason we don't have `management_commands` above defined. Those contain various enqueuing / scheduling mechanisms that we also don't want to run.